pyvo/dal/tap.py

```diff
@@ -501,6 +582,7 @@ def destruction(self, value):
         try:
             response = self._session.post(
                 "{}/destruction".format(self.url),
+                # wondering if this is not IVOA_DATETIME_FORMAT - microsec?
```

Comment:
Yeah seems close - should we try to change this to use your constant?  It does have more precision.  I'm guessing this is listed somewhere in the spec as the format of the time?

Reply:
It turns out both formats (with and without fraction of seconds) are correct so I've added a parser to deal with this. This is temporary, as Python3.7+ already has a method to deal with iso datetime formats.